app/helpers/donors_helper.rb

```diff
@@ -0,0 +1,9 @@
+module DonorsHelper
+  def on_site?(donor)
+    if donor.location.grown_on_site
```

Comment:
Is `Location#grown_on_site` a boolean? 

If so, what do you think about using the `#grown_on_site?` version to communicate that it's a predicate?